docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
         volumes:
             - .:/code
         working_dir: /code
-        command: bash -c "composer install --prefer-source --no-interaction && php ./vendor/bin/phpunit"
```

Comment:
Can we also revert these changes please?

Reply:
@jenssegers, I reverted this changes